commit b12f1f2bbc57a5a488eb754697459ff8b486f43a
Author: Masatake Iwasaki <iwasakims@apache.org>
Date:   Tue Aug 9 12:40:08 2022 +0000

    PHOENIX-6766 Fix failure of sqlline due to conflicting jline dependency pulled from Hadoop 3.3.

diff --git a/pom.xml b/pom.xml
index cbd8b2f24..74fab1b26 100644
--- a/pom.xml
+++ b/pom.xml
@@ -811,6 +811,18 @@
         </exclusions>
         <version>${hadoop.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-yarn-client</artifactId>
+        <version>${hadoop.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.jline</groupId>
+            <artifactId>jline</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+
       <!-- Only for shading in phoenix-server - remove ? -->
       <dependency>
         <groupId>org.apache.hadoop</groupId>
